src/main/java/sda/training/angry_nerds_game/game/MainMenu.java

```diff
@@ -0,0 +1,85 @@
+package sda.training.angry_nerds_game.game;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class MainMenu {
+
+    private final static String MENU = "*-------------------------------------------------*\n"
+            + "*---------------Angry Nerd's v.1.0----------------*\n"
+            + "*-------------------------------------------------*\n"
+            + "*--------------------- MENU ----------------------*\n"
+            + "*-------------------------------------------------*\n"
+            + "1. New Game\n"
+            + "2. Load Game\n"
+            + "3. Game Settings\n"
+            + "4. Quit Game\n"
+            + "*-------------------------------------------------*\n"
+            + "Wybierz 1-4 : ";
+
+
+    public int selectMenu() {
+
+        System.out.print(MENU);
+
+        int opcja = 0;
+        try {
+            opcja = EnterNumber();
+        } catch (InputMismatchException e) {
+            System.out.println("Wprowadź poprawną liczbę.");
+            return 0;
+        }
+        return opcja;
+    }
+
+    public void clearMenu() {
```

Comment:
juz widialem ta funkcje ? skoro musisz robic kopiuj wklej to moze lepiej znalezc dla niej jakies inne miejsce ?